slides/fosdem14/index.html

```diff
@@ -1,5 +1,6 @@
 <section data-markdown class="title center">
   <script type="text/template">
+  <img width="200" src="ucn_logo.png" style="margin-left: 0px; margin-top: 0px; position: fixed" />
```

Comment:
could simplify the `style=...` to `style="margin: 0px"`. not sure that the `position: fixed` is required...?